.copywrite.hcl

```diff
@@ -11,6 +11,7 @@ project {
     "command/asset/*.hcl",
     "command/agent/bindata_assetfs.go",
     "ui/node_modules",
+    "helper/winsvc/strings_*.go",
```

Comment:
just noting that I see this anticipates the merger of #26441 into this branch, since it has some generated files that match this glob.